x-pack/docs/en/rest-api/security/delete-roles.asciidoc

```diff
@@ -24,8 +24,8 @@ Removes roles in the native realm.
 
 The Roles API is generally the preferred way to manage roles, rather than using
```

Comment:
I think this sentence it talking about all the create/update/clear/delete/get API, not just a single API:
```suggestion
The roles APIs are generally the preferred way to manage roles, rather than using
```